alf/environments/metadrive/environments.py

```diff
@@ -35,6 +35,34 @@ class VectorizedTopDownEnv(metadrive.MetaDriveEnv):
 
     """
 
+    @classmethod
+    def default_config(cls) -> metadrive.utils.Config:
+        """The default config is identical to that of the raster TopDownEnv.
+
+        """
+        config = metadrive.MetaDriveEnv.default_config()
+        config["vehicle_config"]["lidar"] = {"num_lasers": 0, "distance": 0}
+        config.update({
+            "frame_skip": 5,
+            "frame_stack": 3,
+            "post_stack": 5,
+        })
+        return config
+
+    def get_single_observation(self, _=None):
+        return VectorizedObservation(self.config["vehicle_config"])
+
+    @property
+    def observation_spec(self):
+        return self.get_single_observation().observation_spec
+
+
+class BirdEyeTopDownEnv(metadrive.MetaDriveEnv):
```

Comment:
The diff is presented in an odd way. Actually the code starting from here to the end of the file is the newly added code. The reason it is presented in such a way is that I removed unrelated options from the `config.update({})` call above.